src/reducer/resource/list/ids.js

```diff
@@ -5,8 +5,17 @@ export default resource => (previousState = [], { type, payload, requestPayload,
         return previousState;
     }
     switch (type) {
-    case CRUD_GET_LIST_SUCCESS:
-        return payload.data.map(record => record.id);
+    case CRUD_GET_LIST_SUCCESS:{
+        if (meta.infiniteList) {
+          if (requestPayload.pagination.page === 1) {
+            return payload.data.map(record => record.id);
+          }
+          return previousState.concat(payload.data.map(record => record.id));
```

Comment:
if (meta.infiniteList || requestPayload.pagination.page === 1) {
  return payload.data.map(record => record.id);
}
return previousState.concat(payload.data.map(record => record.id));

Reply:
it is not this behavior that we want